scripts/cost_script.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+"""
+Converts costs TSV file to summary costs TSV file
+
+Usage: cost_script.py [costs tsv file]
+"""
+#Import modules
+import sys
+import pandas as pd
+import regex as re
+
+file=sys.argv[1]
+
+#initialize list called table where we'll store all the values from tsv
+table = []
+with open(file) as f:
+    for line in f:
+        L = line.split('\t') #split by tab
+        table.append(L)
+
+#delete anything that resembles 'shard' followed by a number. If there's a 3-digit shard at some point, add a re.sub(\d\d\d) line before the re.sub(\d\d) line.
+#have to go in descending order of numerical digits because otherwise will delete shard# and be left with name-of-task# which won't get deleted.
+for i in table:
```

Comment:
Seems like you oughta be able to just capture the shard number with `shard-\d+`, rather than separately implementing one and two digit shards (and comments about three-digits not being supported), right?

Reply:
Ah yes you're right. 
I was hoping there was a more elegant way to do that but wasn't familiar enough with regex. 
But I just tried shard-d/+ and it worked as expected, so I'll change that!